.clang-format

```diff
@@ -0,0 +1,140 @@
+# SPDX-License-Identifier: GPL-2.0
+#
+# clang-format configuration file. Intended for clang-format >= 4.
```

Comment:
Let's  add in this comment:
* I think we must mention that it has been taken from the Linux kernel.
* All changes that have been done comparing with the kernel version.

Reply:
done